client/src/components/comments/index.tsx

```diff
@@ -0,0 +1,112 @@
+'use client';
+
+import { useState } from 'react';
+
+import { cn } from '@/utils';
+import { useUserInfo } from '@/context';
+import { UserIcon, Editor } from '@/components';
+
+import { IComment } from './types';
+import { Comment } from './comment';
+import { EditorBtnGroup } from './btn-group';
+
+export interface ICommentEditor {
+  placeholder: string;
+}
+
+export const CommentEditor = ({ placeholder }: ICommentEditor) => {
```

Comment:
The placeholder prop is not needed in the CommentEditor, since the CommentEditor will receive an array of comments and the taskId.